script.gs

```diff
@@ -70,139 +67,160 @@ function createCard(name, description, labels) {
-      localBody = body.slice(0);
-      localBody = localBody.concat('##Money Requests##\n');
-
-      for (var j = 1; j < 8; j++) {
-        var formatted = Utilities.formatString("**%s**\n %s\n\n", latestItemResponses[i+j].getItem().getTitle(), latestItemResponses[i+j].getResponse());
-        formatted = formatted.concat('\n');
-        localBody = localBody.concat(formatted);
-      }
-      
-      createCard(subject, localBody, [...defaultLabels, '$$$']); // add money label to labels list
-      
-      i = i + 7;
-    }
-    
-    // check for supplies/errands
-    if (latestItemResponses[i].getItem().getTitle() == 'Do you need help getting supplies or running errands?' && latestItemResponses[i].getResponse() == 'Yes') {
-      localBody = body.slice(0);
-      localBody = localBody.concat('##Supplies/Errands##\n');
-
-      for (var j = 1; j < 8; j++) {
-        var formatted = Utilities.formatString("**%s**\n %s\n\n", latestItemResponses[i+j].getItem().getTitle(), latestItemResponses[i+j].getResponse());
-        formatted = formatted.concat('\n');
-        localBody = localBody.concat(formatted);
-      }
-      
-      createCard(subject, localBody, [...defaultLabels, 'Supplies/Errands']); // add supplies label to labels list
-      
-      i = i + 7;
-    }
-    
-    if (latestItemResponses[i].getItem().getTitle() == 'Are you looking for in person childcare and/or online activities for kids and/or petcare?' && latestItemResponses[i].getResponse() == 'Yes') {
-      localBody = body.slice(0);
-      localBody = localBody.concat('##Childcare & Petcare##\n');
-
-      for (var j = 1; j < 12; j++) {
-        var formatted = Utilities.formatString("**%s**\n %s\n\n", latestItemResponses[i+j].getItem().getTitle(), latestItemResponses[i+j].getResponse());
-        formatted = formatted.concat('\n');
-        localBody = localBody.concat(formatted);
-      }
-      
-      createCard(subject, localBody, [...defaultLabels, 'Childcare/Petcare']); // add childcare label to labels list
-
-      i = i + 11;
-    }
-    
-    if (latestItemResponses[i].getItem().getTitle() == 'Are you looking for emotional or spiritual support?' && latestItemResponses[i].getResponse() == 'Yes') {
-      localBody = body.slice(0);
-      localBody = localBody.concat('##Emotional/Spiritual Support##\n');
-
-      for (var j = 1; j < 2; j++) {
-        var formatted = Utilities.formatString("**%s**\n %s\n\n", latestItemResponses[i+j].getItem().getTitle(), latestItemResponses[i+j].getResponse());
-        formatted = formatted.concat('\n');
-        localBody = localBody.concat(formatted);
-      }
-      
-      createCard(subject, localBody, [...defaultLabels, 'Emotional/Spiritual']); // add emotional label to labels list
-      
-      i = i + 1;
-    }
-    
-    if (latestItemResponses[i].getItem().getTitle() == 'Do you need resource support?' && latestItemResponses[i].getResponse() == 'Yes') {
```

Comment:
https://github.com/Mutual-Aid-Medford-and-Somerville/form-to-trello-script/pull/4/commits/4333554a412f9d0f7601a280e3d9f3b80237de27#diff-3ecc0c1cc64672f0f8ab1bef64c251ecR121

The column title changed which caused extra fields to enter the emotional support card.